script/doc-only-change.js

```diff
@@ -0,0 +1,58 @@
+const args = require('minimist')(process.argv.slice(2))
+const octokit = require('@octokit/rest')()
+const path = require('path')
+
+const SOURCE_ROOT = path.normalize(path.dirname(__dirname))
+
+async function checkIfDocOnlyChange () {
+  if (args.prNumber || args.prBranch || args.prURL) {
+    try {
+      let pullRequestNumber = args.prNumber
+      if (!pullRequestNumber || isNaN(pullRequestNumber)) {
+        if (args.prBranch) {
+          // AppVeyor doesn't provide a PR number for branch builds - figure it out from the branch
```

Comment:
appveyor has `APPVEYOR_REPO_BRANCH` that looks to specify the base branch, can we use that?

Reply:
not 100% sure but I don't think that value is always set.

Reply:
Its not the base branch but rather the branch that the changes are on.  See https://ci.appveyor.com/project/electron-bot/electron-ljo26/builds/28165500

Reply:
ugh, https://www.appveyor.com/docs/environment-variables/ says it's the **base** branch, but i guess it's lying